extensions/amp-story-auto-analytics/0.1/auto-analytics-configs.js

```diff
@@ -70,5 +70,35 @@ export const buildGtagConfig = (gtagId) => ({
         'repeat': false,
       },
     },
+    'storyShoppingPLPView': {
+      'on': 'story-shopping-plp-view',
+      'vars': {
+        'event_name': 'custom',
+        'event_action': 'story_shopping_plp_view',
+        'event_category': '${title}',
+        'event_label': 'page id: ${storyPageId}',
```

Comment:
The value of these can just be the variable, in this case it would be `${storyPageId}`.
This follows the pattern of the other `event_label` key value pairs in this file.